server/planning/planning.py

```diff
@@ -55,6 +61,29 @@ def on_deleted(self, doc):
         if 'planning_items' in doc:
             self.delete({'_id': {'$in': doc['planning_items']}})
 
+    def _validate_unique_agenda(self, updates, original):
```

Comment:
`unique` is case-sensitve.  Most of the entities name in Superdesk are case-insensitive. Hence did not use `unique` validator. 

I could have used `iunique` (case-insensitive one) but we are storing planning item in the same collection. We are not currently using `name` for planning item but if start using it or use name field in a different way then we would have the write our own validation. 

I have done a separate PR https://github.com/superdesk/superdesk-core/pull/899 as well so that you can use the `iunique` validator if search backend is elastic.